### PR TITLE
Make `setup.py` version inline

### PR DESCRIPTION
**Pull request by UltraStudioLTD**

### CODE REVIEW

File: pytermgui/__init__.py [hunk 1]
```diff
@@ -36,8 +36,6 @@
 
     warnings.filterwarnings("ignore")
 
-__version__ = "2.1.0"
```
Comment (UltraStudioLTD) [inline pytermgui/__init__.py:39]: Not needed

File: setup.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 setup(
     name="pytermgui",
-    version=pytermgui.__version__,
+    version="2.1.0",
```
Comment (UltraStudioLTD) [inline setup.py:5]: Make `__version__` inline

File: setup.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 setup(
     name="pytermgui",
-    version=pytermgui.__version__,
```
Comment (UltraStudioLTD) [inline setup.py:7]: Remove imported attribute `__version__`

File: setup.py [hunk 4]
```diff
@@ -19,5 +19,7 @@
     url="https://github.com/bczsalba/pytermgui",
     author="BcZsalba",
     author_email="bczsalba@gmail.com",
-    entry_points={"console_scripts": ["ptg = pytermgui.cmd:main"]},
+    entry_points={
+        "console_scripts": ["ptg = pytermgui.cmd:main"]
+    },
```
Comment (UltraStudioLTD) [inline setup.py:22]: Small formatting

**Comment by bczsalba** [comment]
The reason we define `__version__` is to allow other libraries & scripts to check for it. This is not needed.

File: setup.py [hunk 5]
```diff
@@ -1,7 +1,5 @@
 from setuptools import setup
 
-import pytermgui
```
Comment (UltraStudioLTD) [inline setup.py:3]: Remove unwanted import

**Comment by UltraStudioLTD** [comment]
> The reason we define `__version__` is to allow other libraries & scripts to check for it. This is not needed.

Ok

**Pull request closed without merging**